app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApplicationController
+      skip_before_action :verify_authenticity_token
+
+      # rubocop:disable Metrics/AbcSize
+      def create
+        book_suggestion = BookSuggestion.new(editorial: params[:editorial],
```

Comment:
You should use strong parameters instead of doing this.

http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters

You can then add the user to the book_suggestion_params and you will no longer require to disable rubocop.